flow/genericactors.actor.h

```diff
@@ -704,7 +704,7 @@ class AsyncTrigger : NonCopyable {
 // Binds an AsyncTrigger object to an AsyncVar, so when the AsyncVar changes
 // the AsyncTrigger is triggered.
 ACTOR template <class T>
-void forward(Reference<AsyncVar<T> const> from, AsyncTrigger* to) {
+Future<Void> forward(Reference<AsyncVar<T> const> from, AsyncTrigger* to) {
```

Comment:
What was the reason for this change?

Reply:
I think that's probably gap in my Flow understanding. When it was `void` earlier, I wasn't getting a future. Does flow make it run forever by default without a reference? I just added this Future<Void> to make sure that it stays alive once we exit the init() function.

Reply:
An `ACTOR` with return type `void` is uncancellable, so you don't need a handle in that case. I'm not sure what makes sense for this function, but if we change it we have to make sure all other callers properly use it as well.

Reply:
I checked that this is added for the GlobalConfig and is not called at other places.

Reply:
I think we can keep the Future then. Don't see why we should keep it uncancellable.